oeps/processes/oep-0001.rst

```diff
@@ -437,20 +437,15 @@ to capture the decision(s), as deemed appropriate by the Authors and Arbiter.
 To help guide Authors, here are a few ready-made templates that are available
 for use:
 
-* :ref:`PEP-based template <pep_based_template>` based on Python's PEP_ standard.
-* :ref:`ADR-based template <adr_based_template>` based on `Architecture Decision Records`_.
-* :ref:`External link template <external_link_template>` for OEPs with mostly external content.
+* `PEP-based template`_ based on Python's PEP_ standard.
+* `ADR-based template`_ based on `Architecture Decision Records`_.
+* `External link template`_ for OEPs with mostly external content.
 
-..
-  Add a hidden toctree to indicate that these files are referenced from here in the overall
-  table of contents.
```

Comment:
As far as I can tell this never worked - https://open-edx-proposals.readthedocs.io/en/latest/processes/oep-0001.html - I don't see this in the table of contents at the top or the sub-section TOC here: https://open-edx-proposals.readthedocs.io/en/latest/processes/oep-0001.html#oep-structure-and-content